CCDB/src/CCDBDownloader.cxx

```diff
@@ -136,7 +136,54 @@ void closeHandles(uv_handle_t* handle, void* arg)
 void onUVClose(uv_handle_t* handle)
 {
   if (handle != nullptr) {
-    delete handle;
```

Comment:
Shouldn't you simply use malloc / free?

Reply:
Hm, indeed this could also work. I didn't remember that `free` should know how much bytes to delete. Let me try...

Reply:
seems also OK